<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientCrawler</class>
 <widget class="QWidget" name="ClientCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>426</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Client Chasse au Tr√©sor</string>
  </property>
  <widget class="QPushButton" name="pushButtonUp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>262</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fleches.qrc">
     <normaloff>:/img/arrow_up.png</normaloff>
     <normalon>:/img/arrow_up.png</normalon>:/img/arrow_up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>294</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>64</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fleches.qrc">
     <normaloff>:/img/arrow_left.png</normaloff>:/img/arrow_left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>294</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fleches.qrc">
     <normaloff>:/img/arrow_right.png</normaloff>:/img/arrow_right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>326</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fleches.qrc">
     <normaloff>:/img/arrow_down.png</normaloff>:/img/arrow_down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumeroPort">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPortServeur">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
   <property name="value">
    <number>8888</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelAdresseServeur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse du serveur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAdresseServeur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberDistance">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelDistance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformations">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEtat">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Etat</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fleches.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientCrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
